SUMMARY.md

```diff
@@ -157,6 +157,11 @@
 
 ---
 
+* [Advanced Routing](handout/routing-advanced/README.md)
```

Comment:
actually, leave it as 'Guards' not LifeCycle hooks - since the official docs refer to them as guards now.